metadata.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "puppetlabs-ntp",
-  "version": "4.2.0",
+  "version": "6.0.0",
   "author": "Puppet Labs",
   "summary": "Installs, configures, and manages the NTP service.",
   "license": "Apache-2.0",
   "source": "https://github.com/puppetlabs/puppetlabs-ntp",
   "project_page": "https://github.com/puppetlabs/puppetlabs-ntp",
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0 < 5.0.0"}
+    { "name":"puppetlabs/stdlib","version_requirement":">= 4.13.0 < 5.0.0" },
+    { "name":"puppet/tea","version_requirement":">= 0" }
```

Comment:
Talked to @igalic offline and we decided to start pulling those types into stdlib. We'll start out with the easy ones: paths, and URLs, and we'll see from there.

MODULES-3919